src/internal_events/process.rs

```diff
@@ -57,6 +57,70 @@ impl InternalEvent for VectorStopped {
     }
 }
 
+#[derive(Debug)]
+pub struct VectorQuited;
```

Comment:
I believe `VectorQuit` is the correct past tense form, though it's a little awkward.
```suggestion
pub struct VectorQuit;
```